### PR TITLE
Remvoed twitter-widget check

### PR DESCRIPTION
**Pull request by pagachdrunomics**

### CODE REVIEW

File: tests/behat/behat-features/ce-render.feature [hunk 1]
```diff
@@ -70,20 +70,17 @@ Feature: Content is rendered correctly via custom elements
     Then I should see "Drunomics"
     Then I should see "BEHAT: Gallery; Moon or meat?"
     Then I wait for the Instagram paragraph to be rendered
-    And the response should contain "<twitter-widget"
-    Then I should see "We give you top ten reasons why Drupal rocks the CMS arena"
```
Comment (fago) [inline tests/behat/behat-features/ce-render.feature:74]: Why is that removed, does that not work any more?

**Review by fago — CHANGES_REQUESTED** [review]

File: tests/behat/behat-features/ce-render.feature (hunk 1 above)
Comment (pagachdrunomics) [reply tests/behat/behat-features/ce-render.feature:74]: Did not work after adding CE2

**Review by fago — APPROVED** [review]
